src/main/java/org/isf/opd/gui/OpdBrowser.java

```diff
@@ -1038,4 +1118,132 @@ private JButton getFilterButton() {
 		return filterButton;
 	}
 
+	private char getGender() {
+		char sex;
+		if (radioAllGender.isSelected()) {
+			sex = 'A';
+		} else {
+			if (radioMale.isSelected()) {
+				sex = 'M';
+			} else {
+				sex = 'F';
+			}
+		}
+		return sex;
+	}
+	
+	private String getUser() {
+		if (!isSingleUser && radioMyPatients.isSelected()) {
+			return MainMenu.getUser().getUserName();
+		}
+		return null;
+	}
+
+	private char getPatientAttendance() {
+		if (radioAllPatiens.isSelected()) {
+			return 'A';
+		} else if (radioNewAttendance.isSelected()) {
+			return 'N';
+	}
```

Comment:
formatting of the brace looks like it is indented too much to the left

Reply:
done, thanks!